avm/utilities/pipelines/sharedScripts/Get-AvmCsvData.ps1

```diff
@@ -0,0 +1,61 @@
+<#
```

Comment:
@rahalan @AlexanderSehr is this a shared script used by (or planned to be used by) other scripts than the platform ones? Asking because although we exempted the platform folder from the module triggers, adding a script in the sharedScripts folder will again trigger all module workflows

Reply:
@eriqua @AlexanderSehr  It will be a shared script, the moment my two draft PRs are also in place, as they use it as well. Plus those two will introduce another shared script.

Reply:
What about creating a `helper` folder inside the `platform` folder and move this script there?

Reply:
Hey folks +1 on what @eriqua said. It's a bit of an odd one, I agree, but the 'sharedScripts' until now are (I think) shared in between multiple functions, and shared in between functions that are used by the platform- and module- workflows. So if changes occur it makes sense to re-run the workflows.
This script will only be relevant for the platform, so a `helper` inside the platform folder may make the most sense. Thanks @rahalan

Reply:
done

Reply:
@eriqua can we resolve the conversation?